test/Mechanical/translational.jl

```diff
@@ -0,0 +1,58 @@
+import ModelingToolkitStandardLibrary.Blocks
+include(pwd() * "/src/Mechanical/Translational/Translational.jl")
```

Comment:
```suggestion
using Modelingtoolkitstandardlibrary.Mechanical.Translational
```

Reply:
As written, this line returns the error `Translational not defined`. Is this because the Translational module has not been added to the package using the proper procedures (that is, using JL's package manager)? Note that when creating the Translational library, I merely added a new folder to the repo and proceeded to add the requisite lines of code.